engine/gamesys/src/waf_gamesys.py

```diff
@@ -482,6 +482,9 @@ def gofile(self, node):
 new_copy_task('copy prebuilt meshc', '.prebuilt_meshc', '.meshc')
 new_copy_task('copy prebuilt bufferc', '.prebuilt_bufferc', '.bufferc')
 
+# Copy raw png data
+new_copy_task('copy raw png', '.raw', '.rawc')
```

Comment:
Doesn't necessarily have to be a png, so should probably not be named "png"

Reply:
Fixed in: https://github.com/defold/defold/pull/4921/commits/ee8b54705e82bd3f14a3c75619f23bdeaf0f2fa0